### PR TITLE
feat(colorscheme): add neofusion-nvim

### PR DESCRIPTION
**Pull request by 0xricksanchez**

### CODE REVIEW

**Comment by Uzaaft** [comment]
@0xricksanchez  Let's split this into 2 pr's

File: lua/astrocommunity/colorscheme/neofusion-nvim/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# neofusion.nvim
+
+nvim theme blending lava red and ice blue for a vibrant coding experience ✨
+
+**Repository:** <https://github.com/diegoulloao/neofusion.nvim>
+
+```vim
+" Vim Script
+colorscheme neofusion
+```
```
Comment (Uzaaft) [inline lua/astrocommunity/colorscheme/neofusion-nvim/README.md:10]: This can be removed

Comment (mehalter) [reply lua/astrocommunity/colorscheme/neofusion-nvim/README.md:10]: it's fine

**Review by mehalter — APPROVED** [review]
LGTM!

**Pull request merged**